<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opendata</class>
 <widget class="QMainWindow" name="opendata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>공공데이터 청년인턴 목록점검</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>751</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>공공데이터 주소를 입력해주세요</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Url"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_1">
         <property name="text">
          <string>주소입력</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_btn">
         <property name="text">
          <string>초기화</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>공공데이터 제목</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="file_title">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="List_Name">
         <property name="text">
          <string>1. 목록명</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextBrowser" name="result_1"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="BRM">
         <property name="text">
          <string>2. BRM 분류체계</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextBrowser" name="result_2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="Keyword">
         <property name="text">
          <string>3. 키워드 적정성</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextBrowser" name="result_3"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="Data">
         <property name="text">
          <string>4. 실제 데이터 건수</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTextBrowser" name="result_4"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="Extend">
         <property name="text">
          <string>5. 첨부파일 확장자</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QTextBrowser" name="result_5"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
